src/components/LeaderboardCard/LeaderboardCard.tsx

```diff
@@ -0,0 +1,26 @@
+import "./LeaderboardCard.scss"
+import fallbackProfile from "../../../public/assets/images/default-profile-image.png"
+
+type LeaderboardCardProps = {
+    name: string;
+    profileImage: string;
+    totalScore : number;
+};
+
+const LeaderboardCard = ({name, profileImage, totalScore} : LeaderboardCardProps) => {
+    // Using RegEx, it capitalises the first letter of each word in the user's name.
+    const myName = name.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
```

Comment:
This looks like a great piece of code to separate into a function in the utils folder you can then import it and use it in this component.